Bucketeer/Sources/Internal/Utils/Version.swift

```diff
@@ -0,0 +1,3 @@
+enum Version {
+    static let current = "2.0.0"
```

Comment:
I think we will need to add the comment to this line so the CI can update it automatically before releasing it.

E.g.
```swift
static let current = "2.0.0" // x-release-please-version
```

https://github.com/bucketeer-io/ios-client-sdk/blob/main/Bucketeer.podspec#L3

And then add the file path here.

https://github.com/bucketeer-io/ios-client-sdk/blob/main/.github/workflows/release.yml#L28

Reply:
Sorry, I didn't know the release flow.
Thank you, I fixed it.